test/integration/test_analysis_il.c

```diff
@@ -50,8 +50,106 @@ static bool test_analysis_il_vm_step() {
 	mu_end;
 }
 
+static bool test_analysis_il() {
+	RzCore *core = rz_core_new();
+	mu_assert_notnull(core, "init core");
+	RzCoreFile *cf = rz_core_file_open(core, "bins/elf/emulateme.arm64", RZ_PERM_RWX, 0);
+	mu_assert_notnull(cf, "open file");
+	mu_assert("load file", rz_core_bin_load(core, NULL, 0));
+	mu_assert("il vm setup", rz_analysis_il_vm_setup(core->analysis));
+	rz_core_perform_auto_analysis(core, RZ_CORE_ANALYSIS_DEEP);
+
+	RzStrBuf sb = { 0 };
+	RzAnalysisOp op = { 0 };
+
+	// extract and evaluate a single instruction
+	rz_core_analysis_il_reinit(core);
+	rz_analysis_op(core->analysis, &op, core->offset, core->block, core->blocksize, RZ_ANALYSIS_OP_MASK_IL);
+	rz_il_op_effect_stringify(op.il_op, &sb, false);
+	mu_assert_streq(rz_strbuf_get(&sb), "(seq "
+					    "(storew 0 (- (var sp) (bv 64 0x20)) (var x29)) "
+					    "(storew 0 (+ (- (var sp) (bv 64 0x20)) (bv 64 0x8)) (var x30)) "
+					    "(set sp (- (var sp) (bv 64 0x20))))",
+		"stringify il op");
+	mu_assert("eval rzil", rz_core_il_step(core, 1));
+	RzILVal *v = rz_il_vm_get_var_value(core->analysis->il_vm->vm, RZ_IL_VAR_KIND_GLOBAL, "sp");
+	mu_assert_notnull(v, "RzIL vm var value");
+	mu_assert_eq(v->type, RZ_IL_TYPE_PURE_BITVECTOR, "var type");
+	mu_assert_eq(rz_bv_to_ut64(v->data.bv), -0x20, "var value");
+
+	// extract and evaluate a whole function
+	RzAnalysisFunction *f = rz_analysis_get_function_byname(core->analysis, "sym.decrypt");
+	mu_assert_notnull(v, "get function");
+	rz_core_seek(core, f->addr, true);
+	rz_config_set_b(core->config, "io.cache", true);
+	rz_core_cmd_lines(core,
```

Comment:
Ideally, those should be functions too. But if it require too many changes, it can be done in a separate PR.